src/modules/Modal/ModalContent.js

```diff
@@ -1,29 +1,38 @@
-import React, { Component, PropTypes } from 'react'
+import React, { PropTypes } from 'react'
 import classNames from 'classnames'
 import META from '../../utils/Meta'
+import { getUnhandledProps } from '../../utils/propUtils'
 
-export default class ModalContent extends Component {
-  static propTypes = {
-    children: PropTypes.any,
-    className: PropTypes.string,
-  }
-
-  static _meta = {
-    library: META.library.semanticUI,
-    name: 'ModalContent',
-    type: META.type.module,
-    parent: 'Modal',
-  }
-
-  render() {
-    const classes = classNames(
-      this.props.className,
-      'content'
-    )
-    return (
-      <div {...this.props} className={classes}>
-        {this.props.children}
-      </div>
-    )
-  }
+function ModalContent(props) {
+  const { children, className } = props
+
+  const classes = classNames(
+    props.className,
+    'content'
+  )
+
+  const rest = getUnhandledProps(ModalContent, props)
+
+  return (
+    <div className={classes} {...rest}>
+      {children}
+    </div>
+  )
+}
+
+ModalContent._meta = {
+  library: META.library.semanticUI,
```

Comment:
There should be a failing test for this, library was just deprecated today.  Might need to update this branch with the latest master.